src/ng/filter/orderBy.js

```diff
@@ -183,6 +183,7 @@ function orderByFilter($parse) {
     if (sortPredicate.length === 0) { sortPredicate = ['+']; }
 
     var predicates = processPredicates(sortPredicate, reverseOrder);
+    predicates.push({ get: function() { return {}; }, descending: reverseOrder ? -1 : 1});
```

Comment:
I think we should leave a comment explaining this, it won't be very nice to have to look up this issue in the blame for this function to figure out what this means once it stops being clear.

Just a quick header comment with an overview explaining how the predicate works, imho